src/cron/bundle.rs

```diff
@@ -53,7 +53,7 @@ where
     Context: queries::QueryContext + arweave::ArweaveContext<HttpClient>,
     HttpClient: http::Client<Request = reqwest::Request, Response = reqwest::Response>,
 {
-    let arweave = Arweave::new(80, String::from("arweave.net"), String::from("http"));
+    let arweave = Arweave::new(ctx.get_arweave_host());
```

Comment:
As I suggested earlier, looks to me that `Arweave` object should be owned by the context and here you should just ask to borrow it.

Reply:
You might want to have a look what I've done in #25 and do something similar here.

Reply:
Alright, next move is to set arweave in the context